packaging/datadog-agent/source/install_agent.sh

```diff
@@ -51,6 +55,10 @@ if [ ! $apikey ]; then
     exit 1;
 fi
 
+if [ $dd_hostname ]; then
+    printf "\033[31mHOSTNAME is force installed.\033[0m\n"
+fi
```

Comment:
I think you can remove this message entirely: unlike the message for the `DD_API_KEY` var it's not an exit message so I'm not sure it's necessary.